pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "oaklib"
-version = "0.0.0"
+version = "0.1.49"
```

Comment:
was this intentional?

Reply:
yes for testing purposes ... I intend to roll it back